db/migrations/0002_cinemahall_movie_moviesession.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 5.1.2 on 2024-11-07 14:29
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("db", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="CinemaHall",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("rows", models.IntegerField()),
+                ("seats_in_row", models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Movie",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("title", models.CharField(max_length=255)),
+                ("description", models.TextField()),
+                ("actors", models.ManyToManyField(to="db.actor")),
+                ("genres", models.ManyToManyField(to="db.genre")),
```

Comment:
Ensure that the `genre` model exists in the `db` app, as it is referenced here with `ManyToManyField(to="db.genre")`. If it doesn't exist, this will cause an error during migration.